packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -865,59 +877,92 @@ export class GarbageCollector implements IGarbageCollector {
     }
 
     /**
-     * Validates that all new references are correctly identified and processed. The basic principle for validation is
-     * that we should not have new references in the reference graph (GC data) that have not been notified to the
-     * garbage collector via `referenceAdded`.
-     * We validate that the references in the current reference graph should be a subset of the references in the last
-     * run's reference graph + references since the last run.
+     * Finds Missing Routes and logs a Performance event everytime a missing route is found
+     *
+     * Expected Routes = Routes in Current datastore graph and not in Previous datastore graph
+     * New Routes = New routes added by GC when a handle was added or root datastores
+     * Missing Routes = Expected Routes not in New Routes
+     *
+     * Node - represented as nodeId, it's a node on the GC graph
```

Comment:
The distinction between these terms is a good one to add. I think we should add this to the class description as its applicable to the entire class and will help explain what each of these mean.